app/Dockerfile

```diff
@@ -25,6 +29,7 @@ RUN mkdir -p /mattermost/data \
 
 # Configure entrypoint and command
 COPY entrypoint.sh /
+RUN chmod 755 entrypoint.sh
```

Comment:
Still not what we should do. Access mode should be directly on the repository, not an extra-step inside the Dockerfile.
Just remove this line, apply this command on your repository `chmod 755 entrypoint.sh`, commit and push changes :)

Reply:
I did not understand what to do, remove chmod 755 entrypoint.sh?

Reply:
Yes remove this line. But it will then only work if correct mode is apply on `entrypoint.sh` file of the repository.
It worked like that before, but it seems you changed it in your PR.  When I check at your changes I see this : 
![capture d ecran_2017-11-25_17-40-15](https://user-images.githubusercontent.com/8401692/33232498-c17577e8-d207-11e7-9ee4-af20bd90864a.png)
It means that you change mode for the `entrypoint.sh` file from `755` to `644`. You just have to run the command `chmod 755 entrypoint.sh` inside your shell and push changes to fix this.

Reply:
Good 👍